Chapter 5 Transcription

gregmacfarlane opened the pull request:
This one is a little bit longer, but you have two weeks to work on it. This is also the first PR that will have some substantive R work.

gregmacfarlane committed:
Add normal and gumbel PDF / CDF figures

gregmacfarlane committed:
Add figure showing sigmoid choice probabilities

camlunt committed:
merge of text and figures

camlunt committed:
Organized figures 4.1, 4.2, and 4.4 into text

Dr. Macfarlane had added R code for these figures, but my initial commit put his figures in the wrong places. This commit put the three figures where they belong in the text.

gregmacfarlane committed:
Merge branch 'transcribe3' into transcribe4

gregmacfarlane committed:
Add example data loading and log-likelihood

You can take this out, relocate it, or do whatever you think would be valuable.

camlunt committed:
Added equations 4.65 to 4.68

camlunt committed:
Added the three unnumbered tables

camlunt committed:
Added footnotes for Chapter 4

SamKLau committed:
R-chunks for Figure 4.3 and Table 4-1 and 4-2

loganbennett93 committed:
Add tables and equations

I added tables 4-7 and 4-8 as well as equations 4.36-4.42 and 4.55-4.59. I formatted the tables as well as I could. As for Figure 4-7, I don't know where to begin to recreate it.

loganbennett93 committed:
Merge branch 'transcribe4' of https://github.com/byu-transpolab/modechoice into transcribe4

SamKLau committed:
Table and Graphs

SamKLau committed:
Merge branch 'transcribe4' of github.com:byu-transpolab/modechoice into transcribe4

SamKLau committed:
Tables updated

loganbennett93 committed:
Begin formatting greek notation in paragraphs

This was not a task assigned to anyone so I decided to go through and do it.

SamKLau committed:
I tried to knit...

loganbennett93 committed:
Ignore intermediate knit files

loganbennett93 committed:
Merge branch 'transcribe4' of https://github.com/byu-transpolab/modechoice into transcribe4

SamKLau committed:
Merge branch 'transcribe4' of github.com:byu-transpolab/modechoice into transcribe4

SamKLau committed:
hide table and figure code

loganbennett93 committed:
Complete formatting of greek variables in text

loganbennett93 committed:
Take thing away from gitignore that I put in

Merge branch 'transcribe4' of https://github.com/byu-transpolab/modechoice into transcribe4

# Conflicts:
#	04-mnl.Rmd

SamKLau committed:
4.1 equation

barnesmaxmax committed:
Add equations 4.19-4.21

SamKLau committed:
equations

SamKLau committed:
New merged files

Merge branch 'transcribe4' of github.com:byu-transpolab/modechoice into transcribe4

# Conflicts:
#	docs/04-mnl.md
#	modechoice.md

Commit:
Add Tables 4-9 to 4-12

Commit:
test

Commit:
Add Figures

gregmacfarlane committed:
Ignore combined md document

When you build a multiple-document book with bookdown, it creates a merged markdown file and then builds the site from that. There is no reason to keep or commit this file.

gregmacfarlane committed:
Reflow text for easier working in Git

SamKLau committed:
equation changes

Commit:
Add equation 4.43-4.54

Commit:
Add equation 4.69-4.74

Commit:
cleanup libs

bwang0601 committed:
Text Merge

SamKLau committed:
Equations 4.1 to 4.6

SamKLau committed:
Merge branch 'transcribe4' of github.com:byu-transpolab/modechoice into transcribe4

SamKLau committed:
Equations up to 4.12

SamKLau committed:
Equations 4.1 to 4.18

gregmacfarlane committed:
Show how to estimate constants only and null models

SamKLau commented:
Alright, this is intense. But I think this should be good based off everyone's level with programming. Let me know if you disagree and we can adjust. If you have any questions on what can be copy/paste, ask @gregmacfarlane.

@barnesmaxmax 
- [ ] Equations 5.7 to 5.12
- [ ] Figure 5.1
- [ ] Figure 5.4
- [ ] Table 5-8
- [ ] Table 5-9

@bwang0601 
- [ ] Equations 5.13 to 5.18
- [ ] Table 5-2
- [ ] Figure 5.5
- [ ] Table 5-10
- [ ] Table 5-11

@camlunt 
- [ ] Equations 5.19 to 5.23
- [ ] Table 5-1
- [ ] Figure 5-3
- [ ] Table 5-4
- [ ] Figure 5.7
- [ ] Table 5-12

@loganbennett93 
- [ ] Equations 5.24 to 5.34 (plus one on p. 100)
- [ ] Table 5-5 
- [ ] Table 5-6
- [ ] Table 5-7
- [ ] Figure 5.6
- [ ] Table 5-13
- [ ] Figure 5.8

@SamKLau 
- [ ] copy & paste Ch. 5
- [ ] Edit Texts and Formatting
- [ ] Math in Text
- [ ] 2 bibliography references
- [ ] Footnotes (16 to 30)
- [ ] Equations 5.1 to 5.6
- [ ] Figure 5.2 (@gregmacfarlane should this be a copy/paste?)
- [ ] Table 5-3

SamKLau committed:
Added text for Chapter 5

barnesmaxmax committed:
Add equations 4.19-4.35 and 4.60-4.64

Also fixed a bug with repeated text around eq 4.45

SamKLau committed:
added formatting

gregmacfarlane commented:
@SamKLau Figure 5.2 can be copy-pasted.

gregmacfarlane committed:
Add example estimation figure

gregmacfarlane committed:
Clean up likelihood example utility equations.

gregmacfarlane committed:
Change exp to \exp

There is a LaTeX macro that prints an "exp" function rather than just doing three math variables e x p

gregmacfarlane committed:
Build and push website

gregmacfarlane committed:
Deleted trash rds file

gregmacfarlane committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

gregmacfarlane committed:
Add code on conducting tests

barnesmaxmax committed:
Add Table 4.3

Add table 4.3 using KableExtra package.

Commit:
Add Equation 5.13 to 5.18 and their symbols in the text

barnesmaxmax committed:
Tables 4.4-4.6

Add formatted tables for 4.4-4.6.

barnesmaxmax committed:
Add data input fix and equations 5.7-5.12

Added possible data input fix, the data needs to be checked for accuracy still. Input equations 5.7-5.12.

camlunt committed:
Equations 5.19-5.23

camlunt committed:
Fixed some formatting issues in equations 5.13-5.18

barnesmaxmax committed:
Add attempt at indexing SF data

Attempted to index the SF data and to create a choice model to see if indexing was done right.

barnesmaxmax committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

gregmacfarlane committed:
Cleanup data conversion, and show how to make tables for fig 5.1

gregmacfarlane committed:
Table cleanup

SamKLau committed:
Changes in text

SamKLau committed:
Equations 5-1 to 5-6

barnesmaxmax committed:
Add table formatting for figure 5.1

Table formatting added, along with alternate level labels added.

SamKLau committed:
worked on Figure 5.2

SamKLau committed:
math in code

SamKLau committed:
Merge branch 'transcribe5' of github.com:byu-transpolab/modechoice into transcribe5

SamKLau committed:
Table 5-3 and Figure 5-2

SamKLau committed:
added data folder and edits to table 5-3 and figure 5-2

camlunt committed:
Table 5-1 (mostly complete)

Still needs formatting for the percents as well as data for the second column

camlunt committed:
First attempt at Table 5-4

I can't get the coefficients table in summary(base_model) into a useable format (e.g., dataframe)

camlunt committed:
First attempt at Table 5-12 and Figure 5-7

Need understanding of time value of money equation

loganbennett93 committed:
Convert Tables 4-7 and 4-8 into better format

barnesmaxmax committed:
Add Figure 5.3

Add figure 5.3 with confidence intervals.

Commit:
add figure 5.5

Commit:
Table 5-10 incomplete

barnesmaxmax committed:
Add Table 5.8

Add Table 5.8 and add knitr, and kableextra packages to top of script.

loganbennett93 committed:
Equations 5.24 through 5.34

camlunt committed:
Rearranged text-figure order

Figure 5.5 was placed too early in the text; it was moved to be a paragraph later

loganbennett93 committed:
Add equations 5.24 through 5.34

barnesmaxmax committed:
Add start to Table 5-9

loganbennett93 committed:
Add Table 5-5

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

camlunt committed:
Added Figure 5.4

camlunt committed:
Fixed Typos in equation tags

Also committed the second half of the commit called "Rearranged text-figure order"

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

camlunt committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

barnesmaxmax committed:
Re-add figure 5.3

Figure 5.3 got removed during a conflict resolution. I've re-added it here.

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

loganbennett93 committed:
Add Tables 5-6 and 5-13, and figures 5.6 and 5.8

camlunt committed:
Colors for Figure 5-4

camlunt committed:
Table 5-12 and Figure 5.7

gregmacfarlane committed:
Clean up math a bit

gregmacfarlane committed:
Add table and figure references

gregmacfarlane committed:
Rework utility calculations for tables in chapter 4

gregmacfarlane committed:
Rework a couple of tables

barnesmaxmax committed:
Add table 5.9 and fix for table 5.2

gregmacfarlane committed:
Change utility example calculations to be shown with a function that creates a table.

gregmacfarlane committed:
Changes to a couple of equations

gregmacfarlane committed:
Rebuild website

gregmacfarlane committed:
Merge branch 'transcribe4' into transcribe5

loganbennett93 committed:
Add Table 5=7

This table has the correct coefficients and log likelihood values, but has incorrect rho-square values and is missing a number of other statistics given in the table from the original text. I'm not exactly sure how to recreate those statistics.

barnesmaxmax committed:
Edits for Table 5-9

Add equation edits to produce correct data for table 5-9.

gregmacfarlane committed:
Replace likelihood ratio test table with calculations

Note that you can pull the likelihood values directly from the model object.

gregmacfarlane committed:
Minor formatting

I think the basic kable_styling looks good, but the kable_classic has been used. I'll just do a find and replace at some point, so not a big deal. Also, use theme_bw() for the ggplot theme.

gregmacfarlane committed:
Replace constraint with removed variable

You have to have one generic variable in every formula. But instead of giving it a value and constraining it to be one, you can simply pass in a `1` to show that there are no additional variables.

gregmacfarlane committed:
Show how to add digits to the modelsummary output

the `fmt` argument is your friend. ?modelsummary gives a few different methods.

barnesmaxmax committed:
Add table fix for table 5-8

barnesmaxmax committed:
Add decimals to fig 5-8

Commit:
merge

Merge branch 'transcribe5' of https://github.com/byu-transpolab/modechoice into transcribe5

# Conflicts:
#	05-mnl-estimation.Rmd

loganbennett93 committed:
Update Table 5-6 with Calculated LL test values

Commit:
Add table 5-11, error in thrid model, can't find why.

Commit:
Merge branch 'transcribe5' of https://github.com/byu-transpolab/modechoice into transcribe5

Commit:
Adjust Table 5-10

camlunt committed:
Fix Table 5-4

loganbennett93 committed:
Update figure 5-6

camlunt committed:
Table 5-12

camlunt committed:
Table 5-1

SamKLau committed:
Bibliography References

SamKLau committed:
ch. 5 footnotes

gregmacfarlane committed:
Edits to get book.bib to knit

gregmacfarlane committed:
Remove demo log-likelihood with fishing dataset

gregmacfarlane committed:
Minor table cleanup

gregmacfarlane committed:
Work on value-of-time tables

gregmacfarlane committed:
Show market share in percent

gregmacfarlane committed:
Replace references to table 5-2 with tags. Also various text cleanup

gregmacfarlane committed:
Update worktrips file

This is just a change that results when I run the code, but obviously the change hadn't been committed.

gregmacfarlane commented:
A couple more thoughts about this chapter. First, the table numbering got off because we used two tables early in the chapter instead of a figure. We need to do a find-replace operation to get the table numbering to use the bookdown references instead of hard labels.

gregmacfarlane committed:
Provide good example of chunk names and labels

gregmacfarlane committed:
Add cross-references to section headers in chapter 5

gregmacfarlane committed:
Clean up setup chunk printing and comments

gregmacfarlane committed:
Fix multi-level markdown list

gregmacfarlane committed:
Label survey design sections

gregmacfarlane committed:
Correct table references and mild table structure edits for case / alt discussion

gregmacfarlane committed:
Basic cleanup

gregmacfarlane committed:
Simplify trip-alt table

gregmacfarlane committed:
Update numberline image link, knit book
